coverage_comment/coverage.py

```diff
@@ -284,18 +262,9 @@ def get_diff_coverage_info(
         total_num_lines += count_total
         total_num_violations += count_missing
 
-        num_branches_covered = None
-        num_branches_total = None
-        if coverage.meta.branch_coverage:
-            num_branches_covered = 12
-            num_branches_total = 12
-
         percent_covered = compute_coverage(
             num_covered=count_executed,
             num_total=count_total,
-            use_branch_coverage=coverage.meta.branch_coverage,
-            num_branches_covered=num_branches_covered,
-            num_branches_total=num_branches_total,
```

Comment:
I chose _not_ to use the branch data when computing _diff_ coverage even if it's available, which seems like it might be correct. To be honest, I'm not 100% on the theory there. But I noticed that even with branch coverage, each line in the _diff_ will either appear in the `covered_lines` set or the `missing_lines` set. I suppose if a diff line appears in `covered_lines`, but only has partial coverage because it appears in both `executed_branches` and `missing_branches`, then that might adjust the diff numbers down, but I figure a followup PR can add branch-coverage support to file diffs if necessary.

Reply:
branch coverage + diff coverage = 🤯 

At some point, understanding exactly what the number you're looking at means becomes so complex that's it likely not useful nor usable. I think it's the right call.

(it's already hard enough to not confuse the Diff coverage as the coverage rate of the added lines, vs the diff of the coverage, as in how the coverage rate evolved between before and now.)